tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -244,4 +251,34 @@ describe(__filename, () => {
 
     expect(root.find('.SearchFilters-AddonType')).toHaveLength(0);
   });
+
+  it('sets themes filters shelf with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+    const root = render({ _config: fakeConfig });
+    const selectOption = root.find(Select);
+
+    selectOption
```

Comment:
This approach looks better but there's a problem in that the test won't ever fail. This is because the `expect()` call happen in an if branch that may not be reached.

You can do it a couple different ways. Here's one idea:

```js
const optionValues = selectOption.at(1).props().children.map(
  (option) => option.props.value
);
expect(optionValues).toContain(ADDON_TYPE_THEMES_FILTER);
```

There is probably a better way. Also, try to use the [children](http://airbnb.io/enzyme/docs/api/ShallowWrapper/children.html) functions instead because it will make the code more portable among React versions.